src/components/panel/panel.js

```diff
@@ -1012,6 +1146,7 @@ MdPanelRef.prototype.detach = function() {
 
     self.panelContainer.remove();
     self.isAttached = false;
+    self.isOpened = false;
```

Comment:
This should be in close, not detach (unless you move setting it to true to attach instead of open, which I don't recommend). Open and close go together. Attach and detach go together.